docs/docs/deployment/post_deployment/egress_store_approval_process.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
Let's move this file path to `/deployment/reference/egress_store_approval_process.md`.

These steps are not the typical post deployment steps.

Reply:
Fixed. Removed it from Post Deployment. And added it in markdown under Reference.